src/mcp/tool.ts

```diff
@@ -15,6 +15,7 @@ export interface ServerToolContext {
 export interface ServerTool<InputSchema extends ZodTypeAny = ZodTypeAny> {
   mcp: {
     name: string;
+    productGroup?: string;
```

Comment:
Please move this to `_meta`. The `mcp` field is meant to be exactly compatible with MCP protocol. Also call it `feature` not `productGroup`.

Reply:
Sure